cityofaustin/atd-moped
# Prevents dashboard from rerendering on tab switch

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/9349

## Testing
**URL to test:** 
https://deploy-preview-693--atd-moped-main.netlify.app/

**Steps to test:**
Follow a project
Add yourself as a team member of a project
Go to the dashboad
Click betweeb the "following" and "my projects" tabs and note that the table does not rerender when you switch tabs

Example:

https://user-images.githubusercontent.com/35410637/176286256-e73a0ba5-c50e-40bf-bf5e-3144d68e2dec.mov

---
#### Ship list
